svg-validate/svg-validate.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+for filename in "$@"; do
+    if xmllint --schema schemas/svg.xsd --noout $filename; then
```

Comment:
Could we drop the if / then entirely and just run the xmllint line? There's no need for output on success really, pre-commit suppresses it anyway